src/list.js

```diff
@@ -14,28 +14,40 @@ export default class List {
   }
 
   getTailNode() {
-    return this.tail
+    return this.tail.data
   }
 
   contains(stuff) {
-    return this.element = stuff
+    var currentNode = this.head;
+    if(!currentNode) {
+      return false
+    }else{
+      while(currentNode.next!== null) {
+          currentNode = currentNode.next
+          if(currentNode.data === stuff) {
+            return true
+          }else{
+            return false
+          }
+      }
+    }
   }
 
-
   insert(data) {
```

Comment:
Since you a tail pointer you can technically write this in a way that's a little more efficient. 
Essentially, you would:
```
1. point tail.next to the new node
2. assign this.tail to point to the new node
```
very similar to what we did with the `insertFirst()` method, so you don't need to iterate through the list since you have access to the tail. :)